server/src-lib/Hasura/Server/Migrate.hs

```diff
@@ -17,7 +17,7 @@
 module Hasura.Server.Migrate
```

Comment:
There is definitely scope for more refactor here, especially in the case when migrate is required: we end up re-doing many queries like checking for version, schema, extensions, etc.